docs/sql-keywords.md

```diff
@@ -115,6 +115,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>FALSE</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>FETCH</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>FIELDS</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
+  <tr><td>FILTER</td><td>reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
```

Comment:
`FILTER` is reserved in `SQL-2011`. Also, could you please update `TableIdentifierParserSuite`, too.

Reply:
@maropu Thanks for your remind. I will add it.